utils/xmtpRN/conversations.ts

```diff
@@ -134,8 +137,14 @@ async function findDm(args: {
   client: ConverseXmtpClientType;
   peer: string;
   includeSync?: boolean;
+  throwIfNotFound?: boolean;
```

Comment:
Not sure about this pattern 🤔 

Should we just instead trust the function returned type.

Like right now `findDm` is a promise that returns a dm. It's the parent to decide if they want to catch if an error or no no?

We can discuss that in Slack so we can all agree but I think we need to be aligned here

Reply:
Totally agree with you this is a stupid pattern.

I was just adding that because this function already threw below and I didn't want to go trace down anything complicated, but I needed this functionality to return undefined